crypto/params_idx.c

```diff
@@ -0,0 +1,44 @@
+/*
+ *
+ * Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include "internal/e_os.h"
+#include "internal/param_names.h"
+#include <string.h>
+
+struct ossl_param_lookup {
+    int name;
```

Comment:
I would add a comment for `name`, to say that it's the byte position within the gperf generated `ossl_param_lookup_string_pool_contents` where the string can be found.  Otherwise, this "name" is a complete mystery